### ISSUE TITLE
lsif/precise-code-intel server configuration errors are *extremely* verbose

### ISSUE THREAD

**Issue by slimsag** [sourcegraph/sourcegraph#9424]
I have seen this on multiple occasions, in dev environments and in production ones, is there any way we can get this error to be less verbose? I don't see a good reason for this to be more than a single log line.

This can be particularly scary looking when running a server instance ("holy #$%@! that's a lot of errors! my instance must be really broken")

```
13:47:00   postgres_exporter | Terminating postgres_exporter
13:47:00        query-runner | Terminating query-runner
13:47:00        github-proxy | Terminating github-proxy
13:47:00            frontend | Terminating frontend
13:47:00          prometheus | trapped CTRL-C: stopping docker prometheus container
13:47:01          prometheus | 84ef40a7649bc9f06f70b71e0b59ae99c653693d85ab511dee23bb2f4e32906e
13:47:01          prometheus | Error: No such container: prometheus
13:47:01          prometheus | Terminating prometheus
13:47:01           gitserver | Terminating gitserver
13:47:01            searcher | Terminating searcher
13:47:01            replacer | Terminating replacer
13:47:01  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:01  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:01  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:01  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:01  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:01  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:01  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:01  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:01  pci-bundle-manager |     name: 'RequestError',
13:47:01  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:01  pci-bundle-manager |   }
13:47:01  pci-bundle-manager | }
13:47:01        repo-updater | WARN github sync: fetching in batches failed. falling back to sequential fetch, error: Post "http://127.0.0.1:3180/graphql": dial tcp 127.0.0.1:3180: connect: connection refused
13:47:01          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:01          pci-worker |   service: 'precise-code-intel-worker',
13:47:01          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:01          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:01          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:01          pci-worker |       at runMicrotasks (<anonymous>)
13:47:01          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:01          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:01          pci-worker |     name: 'RequestError',
13:47:01          pci-worker |     code: 'ECONNREFUSED'
13:47:01          pci-worker |   }
13:47:01          pci-worker | }
13:47:02  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:02  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:02  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:02  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:02  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:02  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:02  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:02  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:02  pci-bundle-manager |     name: 'RequestError',
13:47:02  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:02  pci-bundle-manager |   }
13:47:02  pci-bundle-manager | }
13:47:04  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:04  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:04  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:04  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:04  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:04  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:04  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:04  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:04  pci-bundle-manager |     name: 'RequestError',
13:47:04  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:04  pci-bundle-manager |   }
13:47:04  pci-bundle-manager | }
13:47:04      pci-api-server | ERROR Failed to retrieve configuration from frontend {
13:47:04      pci-api-server |   service: 'precise-code-intel-api-server',
13:47:04      pci-api-server |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:04      pci-api-server |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:04      pci-api-server |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:04      pci-api-server |       at runMicrotasks (<anonymous>)
13:47:04      pci-api-server |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:04      pci-api-server |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:04      pci-api-server |     name: 'RequestError',
13:47:04      pci-api-server |     code: 'ECONNREFUSED'
13:47:04      pci-api-server |   }
13:47:04      pci-api-server | }
13:47:04          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:04          pci-worker |   service: 'precise-code-intel-worker',
13:47:04          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:04          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:04          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:04          pci-worker |       at runMicrotasks (<anonymous>)
13:47:04          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:04          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:04          pci-worker |     name: 'RequestError',
13:47:04          pci-worker |     code: 'ECONNREFUSED'
13:47:04          pci-worker |   }
13:47:04          pci-worker | }
13:47:05        repo-updater | ERROR Syncer, error: syncer.sync.sourced: 2 errors occurred:
13:47:05        repo-updater | 	* Error getting GitHub repository: nameWithOwner=sourcegraph/sourcegraph: Get "http://127.0.0.1:3180/repos/sourcegraph/sourcegraph": dial tcp 127.0.0.1:3180: connect: connection refused
13:47:05        repo-updater | 	* Get "http://127.0.0.1:3180/user/repos?sort=created&page=1&per_page=100": dial tcp 127.0.0.1:3180: connect: connection refused
13:47:05          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:05          pci-worker |   service: 'precise-code-intel-worker',
13:47:05          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:05          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:05          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:05          pci-worker |       at runMicrotasks (<anonymous>)
13:47:05          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:05          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:05          pci-worker |     name: 'RequestError',
13:47:05          pci-worker |     code: 'ECONNREFUSED'
13:47:05          pci-worker |   }
13:47:05          pci-worker | }
13:47:05             symbols | Terminating symbols
13:47:05               watch | Terminating watch
13:47:05                 web | Terminating web
13:47:05 zoekt-indexserver-0 | Terminating zoekt-indexserver-0
13:47:05             docsite | Terminating docsite
13:47:05      pci-api-server | Terminating pci-api-server
13:47:05             grafana | Terminating grafana
13:47:05        repo-updater | Terminating repo-updater
13:47:07  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:07  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:07  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:07  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:07  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:07  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:07  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:07  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:07  pci-bundle-manager |     name: 'RequestError',
13:47:07  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:07  pci-bundle-manager |   }
13:47:07  pci-bundle-manager | }
13:47:08          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:08          pci-worker |   service: 'precise-code-intel-worker',
13:47:08          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:08          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:08          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:08          pci-worker |       at runMicrotasks (<anonymous>)
13:47:08          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:08          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:08          pci-worker |     name: 'RequestError',
13:47:08          pci-worker |     code: 'ECONNREFUSED'
13:47:08          pci-worker |   }
13:47:08          pci-worker | }
13:47:09  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:09  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:09  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:09  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:09  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:09  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:09  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:09  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:09  pci-bundle-manager |     name: 'RequestError',
13:47:09  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:09  pci-bundle-manager |   }
13:47:09  pci-bundle-manager | }
13:47:10          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:10          pci-worker |   service: 'precise-code-intel-worker',
13:47:10          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:10          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:10          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:10          pci-worker |       at runMicrotasks (<anonymous>)
13:47:10          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:10          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:10          pci-worker |     name: 'RequestError',
13:47:10          pci-worker |     code: 'ECONNREFUSED'
13:47:10          pci-worker |   }
13:47:10          pci-worker | }
13:47:11  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:11  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:11  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:11  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:11  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:11  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:11  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:11  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:11  pci-bundle-manager |     name: 'RequestError',
13:47:11  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:11  pci-bundle-manager |   }
13:47:11  pci-bundle-manager | }
13:47:11  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:11  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:11  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:11  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:11  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:11  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:11  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:11  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:11  pci-bundle-manager |     name: 'RequestError',
13:47:11  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:11  pci-bundle-manager |   }
13:47:11  pci-bundle-manager | }
13:47:12          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:12          pci-worker |   service: 'precise-code-intel-worker',
13:47:12          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:12          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:12          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:12          pci-worker |       at runMicrotasks (<anonymous>)
13:47:12          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:12          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:12          pci-worker |     name: 'RequestError',
13:47:12          pci-worker |     code: 'ECONNREFUSED'
13:47:12          pci-worker |   }
13:47:12          pci-worker | }
13:47:12  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:12  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:12  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:12  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:12  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:12  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:12  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:12  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:12  pci-bundle-manager |     name: 'RequestError',
13:47:12  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:12  pci-bundle-manager |   }
13:47:12  pci-bundle-manager | }
13:47:13          pci-worker | ERROR Failed to retrieve configuration from frontend {
13:47:13          pci-worker |   service: 'precise-code-intel-worker',
13:47:13          pci-worker |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:13          pci-worker |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:13          pci-worker |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:13          pci-worker |       at runMicrotasks (<anonymous>)
13:47:13          pci-worker |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:13          pci-worker |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:13          pci-worker |     name: 'RequestError',
13:47:13          pci-worker |     code: 'ECONNREFUSED'
13:47:13          pci-worker |   }
13:47:13          pci-worker | }
13:47:14  pci-bundle-manager | ERROR Failed to retrieve configuration from frontend {
13:47:14  pci-bundle-manager |   service: 'precise-code-intel-bundle-manager',
13:47:14  pci-bundle-manager |   error: GotError: connect ECONNREFUSED 127.0.0.1:3090
13:47:14  pci-bundle-manager |       at onError (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:140:29)
13:47:14  pci-bundle-manager |       at handleRequest (/Users/slimsag/go/sourcegraph/cmd/lsif-server/node_modules/got/dist/source/request-as-event-emitter.js:173:17)
13:47:14  pci-bundle-manager |       at runMicrotasks (<anonymous>)
13:47:14  pci-bundle-manager |       at processTicksAndRejections (internal/process/task_queues.js:93:5)
13:47:14  pci-bundle-manager |       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:14) {
13:47:14  pci-bundle-manager |     name: 'RequestError',
13:47:14  pci-bundle-manager |     code: 'ECONNREFUSED'
13:47:14  pci-bundle-manager |   }
13:47:14  pci-bundle-manager | }
13:47:16               caddy | Terminating caddy
Password:
13:47:16      syntect_server | Terminating syntect_server
13:47:16 zoekt-indexserver-1 | Terminating zoekt-indexserver-1
13:47:16   zoekt-webserver-1 | Terminating zoekt-webserver-1
13:47:16              jaeger | {"level":"info","ts":1585687636.6875598,"caller":"flags/service.go:145","msg":"Shutting down"}
13:47:16              jaeger | {"level":"info","ts":1585687636.687644,"caller":"healthcheck/handler.go:128","msg":"Health Check state change","status":"unavailable"}
13:47:16              jaeger | {"level":"info","ts":1585687636.687668,"caller":"app/agent.go:89","msg":"shutting down agent's HTTP server","addr":"[::]:5778"}
13:47:16              jaeger | {"level":"info","ts":1585687636.687742,"caller":"app/agent.go:73","msg":"agent's http server exiting"}
13:47:16              jaeger | {"level":"info","ts":1585687636.6881258,"caller":"healthcheck/handler.go:128","msg":"Health Check state change","status":"unavailable"}
13:47:16              jaeger | {"level":"info","ts":1585687636.688267,"caller":"flags/service.go:153","msg":"Shutdown complete"}
13:47:16              jaeger | Terminating jaeger
13:47:16  pci-bundle-manager | Terminating pci-bundle-manager
13:47:16          pci-worker | Terminating pci-worker
```

**Comment by efritz** [comment]
Will look into this tonight.

**Comment by efritz** [comment]
Fixed in #9474.
